src/interfaces.ts

```diff
@@ -12,6 +12,25 @@ export type ValidationRule = <T = any>(prop: keyof T, errorMessage: string, attr
     ErrorMessage
 ];
 
+export interface IBindInputResult<T = any> {
```

Comment:
I'm not in love with the name if you have any suggestions

Reply:
`IBoundInput` ?

Reply:
works for me :)